docs/general/metrics.md

```diff
@@ -89,6 +90,27 @@ When introducing a new metric name check all existing schema files to make sure
 the name does not appear as a key of any "rename_metrics" section (keys denote
 old metric names in rename operations).
 
+### Metric attributes
+
+Metric attributes SHOULD follow the general [attribute naming rules](attribute-naming.md).
+In particular, metric attributes MUST have a namespace.
+
+Metric attributes SHOULD be added under the metric namespace when their usage and
+semantics are exclusive to the metric.
+
+Examples:
+
+Attributes `mode` and `mountpoint` for metric `system.filesystem.usage`
+should be namespaced as `system.filesystem.mode` and `system.filesystem.mountpoint`.
+
+Metrics can also have attributes outside the metric namespace when these come
+from the attributes registry.
```

Comment:
"when these come from the attributes registry" sort of makes it sounds like otherwise they don't need to be in the attributes registry

Reply:
Yeah I don't love it as well.. but I thought it would be good to point that there could be metrics like that and to remove the possible confusion of "why these are not namespaced under the metric". Do you have any suggestion? 

I removed that portion, now it's just:

> Metrics can also have attributes outside of its namespace.